lib/resources/docker.rb

```diff
@@ -203,8 +203,13 @@ def parse_json_command(labels, subcommand)
         # does not include a slash since that is not a valid character in a container name
         j['names'] = j['names'].split(',').find { |c| !c.include?('/') } if j.key?('names')
 
+        # Split labels on ','
+        # This allows `its('labels') { should include 'foo=bar' }` to work
+        j['labels'] = j['labels'].split(',') if j.key?('labels')
```

Comment:
I encourage you to think of the [Boy Scout Rule](https://deviq.com/boy-scout-rule/).  Consider replacing the `j`, `k`, and `v` variables with names that are meaningful to humans and amenable to grepping.

Reply:
Can do!